build-in-container/scripts/setup.sh

```diff
@@ -59,6 +59,36 @@ check_specs() {
     fi
 }
 
+# setup custom repo to install RPMs from
+setup_custom_repo() {
+    echo "------------ Setting up custom repo ------------"
+    if [[ ! -z "${RPM_repo}" ]]; then
+        echo "------------ RPM_repo is $RPM_repo ------------"
+        cp $MARINER_BASE_DIR/scripts/custom-repo.repo $MARINER_BASE_DIR/toolkit/repos/
+        sed -i "s~<CUSTOM_REPO_BASE_URL>~${RPM_repo}~" $MARINER_BASE_DIR/toolkit/repos/custom-repo.repo
+        cat $MARINER_BASE_DIR/toolkit/repos/custom-repo.repo >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
+    fi
+
+    if [[ ! -z "${RPM_storage}" ]]; then
+        echo "------------ RPM_storage is $RPM_storage ------------"
+        #install azcopy
+        wget -O azcopy_v10.tar.gz https://aka.ms/downloadazcopy-v10-linux  || { echo "ERROR: Could not install azcopy"; exit 1; }
```

Comment:
This seems brittle as is. Is this the proper way to get `azcopy`? And will the tarball always be `azcopy_v10.tar.gz`?

Reply:
I got this from here https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10#download-azcopy They told to use specific version as azcopy behavior might change across versions

Reply:
Thanks for the pointer! Not a fan of how AzCopy is doing this, but it is the officially documented way of getting it, and it sounds like they're only guaranteeing UI compat within versions. Makes sense.